pkg/cosign/keys.go

```diff
@@ -38,10 +38,11 @@ import (
 )
 
 const (
-	pemType  = "ENCRYPTED COSIGN PRIVATE KEY"
-	sigkey   = "dev.cosignproject.cosign/signature"
-	certkey  = "dev.sigstore.cosign/certificate"
-	chainkey = "dev.sigstore.cosign/chain"
+	pemType   = "ENCRYPTED COSIGN PRIVATE KEY"
+	sigkey    = "dev.cosignproject.cosign/signature"
+	certkey   = "dev.sigstore.cosign/certificate"
+	chainkey  = "dev.sigstore.cosign/chain"
+	BundleKey = "dev.sigstore.cosign/bundle"
```

Comment:
This name looks fine to me, anyone else have an opinion?

Reply:
we could do `dev.sigstore.cosign/bundle/v1` if we plan on versioning this

Reply:
> we could do dev.sigstore.cosign/bundle/v1 if we plan on versioning this

Could also add a `.v2` later if that ever happens :p